src/objects/entities/Ben3.ts

```diff
@@ -119,12 +119,39 @@ class Ben3 extends Entity {
     }
   }
 
+  left() {
+    const rot = this.gameObject.rotation - Phaser.Math.DegToRad(5);
+    this.gameObject.setRotation(rot);
+    this.head.body.mass = 0;
+    // this.gameObject.setAngle(this.gameObject.angle - 5);
+  }
+
+  right() {
+    const rot = this.gameObject.rotation + Phaser.Math.DegToRad(5);
+    this.gameObject.setRotation(rot);
+    // this.head.body.mass = 0;
+    // this.gameObject.setAngle(this.gameObject.angle + 5);
+  }
+
   update(time: number, delta: number) {
     super.update(time, delta);
 
+    const AkeyDown = this.scene.input.keyboard?.addKey(
+      Phaser.Input.Keyboard.KeyCodes.A,
+    ).isDown;
+    if (AkeyDown) {
+      this.left();
+    }
+    const DkeyDown = this.scene.input.keyboard?.addKey(
+      Phaser.Input.Keyboard.KeyCodes.D,
+    ).isDown;
+    if (DkeyDown) {
+      this.right();
+    }
```

Comment:
how are we gonna do this on mobile? :thinking:

Reply:
If we're not going to on screen buttons then either left/right swiping or touching left/right side of the screen.